api/v1beta1/hyperconverged_types.go

```diff
@@ -856,6 +856,22 @@ type NodeInfoStatus struct {
 	WorkloadsArchitectures []string `json:"workloadsArchitectures,omitempty"`
 	// ControlPlaneArchitectures is a distinct list of the CPU architecture of the control-plane nodes.
 	ControlPlaneArchitectures []string `json:"controlPlaneArchitectures,omitempty"`
+	// RecommendedCpuModels is a list of recommended CPU models for the cluster based on available nodes
+	RecommendedCpuModels []CpuModelInfo `json:"recommendedCpuModels,omitempty"`
+}
+
+// CpuModelInfo contains information about a CPU model and its availability in the cluster
+type CpuModelInfo struct {
```

Comment:
All the fields in this struct are mandatory, because there in no "emitempty" directive for them.

Is that intentional?